xdsl/interpreters/cf.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+from xdsl.dialects import cf
+
+from xdsl.interpreter import (
+    Interpreter,
+    InterpreterFunctions,
+    Successor,
+    impl_terminator,
+    register_impls,
+)
+
+
+@register_impls
+class CfFunctions(InterpreterFunctions):
+    @impl_terminator(cf.Branch)
+    def run_br(self, interpreter: Interpreter, op: cf.Branch, args: tuple[Any, ...]):
+        return Successor(op.successor, args), ()
+
+    @impl_terminator(cf.ConditionalBranch)
+    def run_cond_br(
+        self, interpreter: Interpreter, op: cf.ConditionalBranch, args: tuple[Any, ...]
+    ):
+        cond: int = args[0]
```

Comment:
```suggestion
        cond: bool = args[0]
```

Why not like this?

Reply:
condition is `i1` in cf, but I guess in Python-land we can cast it?

Reply:
fun fact :)
```
>>> issubclass(bool, int)
True
```

Reply:
So it really depends on whether you philosophically think of i1 as bool or int, at runtime it's still going to be an instance of bool there's no actual casting going on

Reply:
Ah okay, that's fine then.